.travis.yml

```diff
@@ -1,58 +0,0 @@
-dist: focal
-language: python
-cache: pip
-
-notifications:
-  # will post a summary as comment on github in case of failure
-  webhooks: https://www.travisbuddy.com/
-  on_success: never
-
-services:
-  - postgresql
-
-addons:
-  postgresql: "13"
-
-matrix:
-  include:
-    - { python: "3.6", env: TOXENV=py36-django22-djangorestframework310 }
-    - { python: "3.7", env: TOXENV=py37-django22-djangorestframework310 }
-    - { python: "3.8", env: TOXENV=py38-django22-djangorestframework310 }
-    - { python: "3.9", env: TOXENV=py39-django22-djangorestframework310 }
-    - { python: "3.6", env: TOXENV=py36-django22-djangorestframework312 }
-    - { python: "3.7", env: TOXENV=py37-django22-djangorestframework312 }
-    - { python: "3.8", env: TOXENV=py38-django22-djangorestframework312 }
-    - { python: "3.9", env: TOXENV=py39-django22-djangorestframework312 }
-    - { python: "3.6", env: TOXENV=py36-django30-djangorestframework312 }
-    - { python: "3.7", env: TOXENV=py37-django30-djangorestframework312 }
-    - { python: "3.8", env: TOXENV=py38-django30-djangorestframework312 }
-    - { python: "3.9", env: TOXENV=py39-django30-djangorestframework312 }
-    - { python: "3.6", env: TOXENV=py36-django31-djangorestframework312 }
-    - { python: "3.7", env: TOXENV=py37-django31-djangorestframework312 }
-    - { python: "3.8", env: TOXENV=py38-django31-djangorestframework312 }
-    - { python: "3.9", env: TOXENV=py39-django31-djangorestframework312 }
-
-branches:
-  only:
-    - master
-
-before_install:
-  - sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable -y
-  - yes | sudo sh /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh
```

Comment:
is this not necessary in github actions?

Reply:
Yes as we are using postgres from github services, we don't need to install it